--- PTlink6.19.4/autoconf/configure.in	2005-08-23 19:06:36.000000000 +0300
+++ PTlink6.19.4.link/autoconf/configure.in	2006-04-15 14:59:56.486318551 +0300
@@ -31,13 +31,15 @@
 AC_PROG_INSTALL
 
 dnl Checks for libraries.
-dnl Replace `main' with a function in -lnsl:
-dnl Replace ain' with a function in -lnsl:
-AC_CHECK_LIB(nsl, gethostbyname)
-AC_CHECK_FUNC(res_mkquery,, [
-  AC_CHECK_LIB(resolv, res_mkquery, LIBS="-lresolv")
+AC_SEARCH_LIBS(gethostbyname,nsl)
+AC_CHECK_FUNC(res_mkquery,,[
+  AC_CHECK_FUNC(__res_mkquery,,[
+    AC_CHECK_LIB(resolv, res_mkquery,,[
+      AC_CHECK_LIB(resolv, __res_mkquery)
+    ])
+  ])
 ])
-AC_CHECK_LIB(socket, socket)
+AC_SEARCH_LIBS(socket, socket)
 AC_CHECK_FUNC(crypt,, AC_CHECK_LIB(descrypt, crypt,,AC_CHECK_LIB(crypt, crypt,,)))
 AC_CHECK_FUNC(inet_ntoa,, AC_CHECK_LIB(nsl, inet_ntoa,,))
 AC_CHECK_FUNC(inet_aton,, AC_CHECK_LIB(resolv, inet_aton,,))
