diff -ur PTlink6.15.1-orig/autoconf/configure.in PTlink6.15.1/autoconf/configure.in
--- PTlink6.15.1-orig/autoconf/configure.in	2003-04-08 11:48:00.000000000 -0600
+++ PTlink6.15.1/autoconf/configure.in	2003-08-16 21:33:06.794771625 -0600
@@ -295,11 +295,11 @@
 
   IRCd version:                        ${VERSION}
   Default data path:                   ${prefix}
-  Binary path:                         ${prefix}/ircd
+  Binary path:                         ${sbindir}/ircd
   Compiler:                            ${CC}
 
   Paths set at include/path.h
 "
-echo "#define SPATH \"${prefix}/ircd\"" > include/path.h
-echo "#define DPATH \"${prefix}\"" >> include/path.h
+echo "#define SPATH \"${sbindir}/ircd\"" > include/path.h
+echo "#define DPATH \"${sysconfdir}/\"" >> include/path.h
 
diff -ur PTlink6.15.1-orig/include/config.h PTlink6.15.1/include/config.h
--- PTlink6.15.1-orig/include/config.h	2003-07-04 11:57:08.000000000 -0600
+++ PTlink6.15.1/include/config.h	2003-08-16 21:32:27.325741670 -0600
@@ -101,14 +101,14 @@
  *
  */
 
-#define CPATH   "ircd.conf"
-#define KPATH   "kline.conf"
-#define DLPATH  "kline.conf"
-#define MPATH   "ircd.motd"
-#define LPATH   "ircd.log"
-#define PPATH   "ircd.pid"
-#define OPATH   "opers.motd"
-#define WPATH   "webchat.motd"
+#define CPATH   "/etc/ircd/ircd.conf"
+#define KPATH   "/etc/ircd/kline.conf"
+#define DLPATH  "/etc/ircd/kline.conf"
+#define MPATH   "/etc/ircd/ircd.motd"
+#define LPATH   "/var/log/ircd/ircd.log"
+#define PPATH   "/var/run/ircd.pid"
+#define OPATH   "/etc/ircd/opers.motd"
+#define WPATH   "/etc/ircd/webchat.motd"
 
 
 /*
@@ -504,8 +504,8 @@
  * define IRC_UID to that UID.  This should only be defined if you are running
  * as root and even then perhaps not.
  */
-/* #define IRC_UID 1001 */
-/* #define IRC_GID 31 */
+#define IRC_UID 75
+#define IRC_GID 75
 
 /* CLIENT_FLOOD - client excess flood threshold
  * this controls the number of bytes the server will allow a client to
